content/concepts/asset-manager.md

```diff
@@ -0,0 +1,2126 @@
+
+::: warning
+The `attribute`, `locale` and `channel` properties are mandatory. 
+If the `MEDIA_FILE_ATTRIBUTE_NAME` is the code of an attribute that is not localizable, the `locale` property should be set to `null`.  
+If the `MEDIA_FILE_ATTRIBUTE_NAME` is the code of an attribute that is not scopable, the `channel` property should be set to `null`. 
+:::
+
+::: info
+There are additional business rules regarding this `target` property whenever you have multiple transformations for the same asset family. See the [Dealing with several transformations](#dealing-with-several-transformations) section for more details.
+:::
+
+### Target file
+
+The `target` property allows you to define in which attribute value the PIM should generate the new variation.
+
+It follows this format:
+```json
+{
+  "target": {
+    "attribute": MEDIA_FILE_ATTRIBUTE_NAME,
+    "locale": TARGET_LOCALE_CODE,
+    "channel": TARGET_CHANNEL_CODE
+  },...
+}
+```
+
+In this formula:
+ - `MEDIA_FILE_ATTRIBUTE_NAME` is the code of the asset attribute that holds the target file. This asset attribute should be defined in the asset family and of type `media_file`.
+ - `TARGET_LOCALE_CODE` is an existing locale code when `MEDIA_FILE_ATTRIBUTE_NAME` is the code of a localizable asset attribute.
+ - `TARGET_CHANNEL_CODE` is an existing channel code when `MEDIA_FILE_ATTRIBUTE_NAME` is the code of a scopable asset attribute.
+
+::: warning
+The `attribute`, `locale` and `channel` properties are mandatory. 
+If the `MEDIA_FILE_ATTRIBUTE_NAME` is the code of an attribute that is not localizable, the `locale` property should be set to `null`.  
+If the `MEDIA_FILE_ATTRIBUTE_NAME` is the code of an attribute that is not scopable, the `channel` property should be set to `null`. 
+:::
+
+::: info
+There are additionnal business rules regarding this `target` property whenever you have multiple transformations for the same asset family. See the [Dealing with several transformations](#dealing-with-several-transformations) section for more details.
+:::
+
+### Target filename
+
+You can give a name to the generated target file. By default, the naming is based on the filename of the source file. You can define a suffix and/or a prefix that will be concatenated to this filename and be used as the filename of the target file. 
+
+To do this, use:
+- the `filename_prefix` property, the string that will be prepend to the source filename.
+- the `filename_suffix` property, the string that will be append to the source filename.
+
+You can use both properties if you want to suffix and prefix the source filename.
+
+::: warning
+You need to define at least either a suffix or a prefix, as the target filename can't be the same as the source filename.
+:::
+
+::: info
+Suffix and prefix can only contain alphanumeric characters and the following characters: `_`, `-`, `.` and space.
```

Comment:
```suggestion
Suffix and prefix can only contain alphanumeric characters, the following characters: `_`, `-`, `.` and space.
```

Reply:
The cited characters are not alphanumeric. So you can use alphanumeric characters AND those characters.